test/snowpack/protocol_test.exs

```diff
@@ -31,7 +31,15 @@ defmodule Snowpack.ProtocolTest do
     end
 
     @tag :capture_log
-    test "with connection_closed error query is retried ONLY once", %{pid: pid} do
+    test "with connection_closed error query is retried 3 times", %{pid: pid} do
+      expect Snowpack.ODBC, :query, fn _pid, _statement, _params, _opts, _with_query_id ->
+        {:error, Snowpack.Error.exception(:connection_closed)}
+      end
```

Comment:
I think if we do this we can eliminate the extra expects. :)

```suggestion
      expect Snowpack.ODBC, :query, 3, fn _pid, _statement, _params, _opts, _with_query_id ->
        {:error, Snowpack.Error.exception(:connection_closed)}
      end
```

Reply:
Forgot about that. :)